sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -493,6 +493,67 @@ public static void endSession() {
     getCurrentHub().endSession();
   }
 
+  /**
+   * Captures the transaction and enqueues it for sending to Sentry server.
+   *
+   * @param transaction the transaction
+   * @param hint the hint
+   * @return transaction's id
+   */
+  public static SentryId captureTransaction(SentryTransaction transaction, Object hint) {
```

Comment:
```suggestion
  public static @Nullable SentryId captureTransaction(final @NotNull SentryTransaction transaction, final @Nullable Object hint) {
```

Reply:
missing annotations for the other methods too, either on params or return type

Reply:
👍